### DISCUSSION TITLE
Language selection from videos to download

### DISCUSSION THREAD

**Discussion by manu-p** [subhra74/xdm#564] [Q&A]
Hi,

Trying for the 1st time to use xdman on Ubuntu 20.04, looks pretty good!

One thing I could not solve so far is, trying to download a video from https://www.arte.tv/fr/ I always get a german version while I'm french and see the streaming in french.
I found I could choose between different video qualities to download but all of them are coming in german. And VLC shows no choice for audio track from the downloaded mp4 video...

Is there a way to select the language for the video to download thru xdman?

**Comment by subhra74** [comment]
This feature will be added in next version

**Reply by manu-p** [reply]
Good! thanks @subhra74
